pkg/resources/validation/cluster/provisioningcluster.go

```diff
@@ -23,6 +23,7 @@ import (
 const globalNamespace = "cattle-global-data"
 
 var mgmtNameRegex = regexp.MustCompile("^c-[a-z0-9]{5}$")
```

Comment:
Interesting that the expected format of a creating a management cluster in the api is different than the one that is created as a by product of creating a provisioning cluster. Is that by design?

Reply:
Yes it is, my understanding is that the different naming conventions allow us to determine if a cluster is legacy or not.